hardhat.config.ts

```diff
@@ -1,38 +1,42 @@
 import { HardhatUserConfig } from "hardhat/config";
 
+import "@matterlabs/hardhat-zksync-node";
 import "@matterlabs/hardhat-zksync-deploy";
 import "@matterlabs/hardhat-zksync-solc";
-
 import "@matterlabs/hardhat-zksync-verify";
 
-// dynamically changes endpoints for local tests
-export const zkSyncTestnet =
-  process.env.NODE_ENV == "test"
-    ? {
-        url: "http://127.0.0.1:8011",
-        ethNetwork: "http://127.0.0.1:8045",
-        zksync: true,
-      }
-    : {
-        url: "https://testnet.era.zksync.dev",
-        ethNetwork: "goerli",
-        zksync: true,
-        // contract verification endpoint
-        verifyURL:
-          "https://zksync2-testnet-explorer.zksync.dev/contract_verification",
-      };
-
 const config: HardhatUserConfig = {
-  zksolc: {
-    version: "latest",
-    settings: {},
-  },
-  defaultNetwork: "zkSyncTestnet",
+  defaultNetwork: "dockerizedNode",
   networks: {
+    zkSyncEraTestnet: {
+      url: "https://testnet.era.zksync.dev",
+      ethNetwork: "goerli",
+      zksync: true,
+      verifyURL: "https://zksync2-testnet-explorer.zksync.dev/contract_verification",
+    },
+    zkSyncEraMainnet: {
+      url: "https://mainnet.era.zksync.io",
+      ethNetwork: "mainnet",
+      zksync: true,
+      verifyURL: "https://zksync2-mainnet-explorer.zksync.io/contract_verification",
+    },
+    dockerizedNode: {
+      url: "http://localhost:3050",
+      ethNetwork: "localhost",
+      zksync: true,
+    },
+    inMemoryNode: {
+      url: "http://localhost:8011",
+      ethNetwork: "localhost",
```

Comment:
nit: Should we remove `ethNetwork` for `inMemoryNode` to avoid confusion?

Reply:
When I remove it, hardhat fails with an error that `ethNetwork` should be present

Reply:
...Darn... ok, leave for now 🤷